### PR TITLE
added texts and login arrow for more clarity

### PR DESCRIPTION
**Pull request by GijsWeterings**

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -68,6 +68,8 @@
       </div>
     </div>
 
+    <img src="images/loginarrow.png" class="loginarrow" alt="loginarrow" ng-if="!loggedIn">
```
Comment (JeremybellEU) [inline app/index.html]: Shouldn't this be `ng-hide="loggedIn"`?

Comment (GijsWeterings) [inline app/index.html]: actually no ;) ng-if deletes the DOM node when the value is false, so once logged in we don't have to keep the image in RAM

**Comment by Jaapp-** [comment]
LGTM

**Comment by JWGmeligMeyling** [comment]
* Rebased on master
* Hide arrow on smaller screens
* Show descriptive messages on first use

Ready to merge!

**Comment by Azkle** [comment]
I'll check it out and see how it looks.

**Comment by JWGmeligMeyling** [comment]
I fixed that by only showing the text the first time. I still think we can improve by only showing the most required text, because most users will only scroll quickly through the text. But this is another story ;)

**Comment by Azkle** [comment]
Looks good, only one remark.
On ctrl-f5 the login with soundcloud arrow image is page filling and then it dissapears and makes place for the normal page. Is there something small we can do about that?

File: app/index.html [hunk 2]
```diff
@@ -71,6 +71,8 @@
       </div>
     </div>
 
+    <img src="images/loginarrow.png" class="loginarrow visible-lg-block" alt="loginarrow" ng-if="!loggedIn">
```
Comment (JWGmeligMeyling) [inline app/index.html]: Add `width` and `height` attributes, add `style="display: none;" and then let angular make it appear using `ng-show="loggedIn"`

Comment (Azkle) [inline app/index.html]: ng-hide="loggedin" I suposse, we only want to show it when not logged in.

Comment (JWGmeligMeyling) [inline app/index.html]: this is correct. Ng-if removes the element from the DOM, which is preferable. Ng-if requires a negation.

**Pull request merged**